src/applications/data/importManager.tsx

```
@@ -180,7 +194,21 @@ const useApps = (): TaskType[] => {
           "setUpWidth" : 850,
           "setUpHeight" : 500,
         }
+      },
+    {
+      "component": <Suspense fallback={null}><Alert icon="" text={<></>} onClick={() => undefined}/></Suspense>,
+      "type": "App",
+      "id": 2229,
```

Biome (1.9.4)

[error] Avoid using unnecessary Fragment.

A fragment is redundant if it contains only one child, or if it is the child of a html element, and is not a keyed fragment.

(lint/complexity/noUselessFragments)